source/indexes.txt

```diff
@@ -116,6 +116,32 @@ input.
    |compass-short| shows the type of your new index as
    :guilabel:`Wildcard`.
 
+.. _hide-index:
+
+Hide an Index
```

Comment:
[question]

Does Compass provide a way to unhide an index as well? If so, maybe we should mention that somewhere in this section.

Reply:
Yes! It's the same procedures aside from the changes in hover text. Will note that.